### ISSUE TITLE
Interceptors incompatibility introduced in 1.1.0

### ISSUE THREAD

**Issue by MilesTFox** [axios/axios#5040]
Describe the bug
Started getting the following when upgraded to axios latest version.

To Reproduce
Use latest version of axios in package.json, alongside axios-retry.

```
 /data/api/node_modules/axios-retry/lib/cjs/index.js:263
   axios.interceptors.request.use(function (config) {
                      ^

 TypeError: Cannot read properties of undefined (reading 'request')
     at axiosRetry (/data/api/node_modules/axios-retry/lib/cjs/index.js:263:22)
```

Expected behavior
In an ideal world, the nodejs app would have started without error.

Reverting to 1.0.0, functions properly.

Environment
Axios Version 1.1.0
Additional context/Screenshots
N/A

**Comment by addisonhardy** [comment]
I believe this issue and #5038 share the same problem - `axios.interceptors` is not available on the `axios` object in v1.1.0, but it is available via `axios.default.interceptors`.

**Comment by jasonsaayman** [comment]
Hi :wave:

Please try the latest pre-release by running the following:

```bash
npm i axios@1.2.0-alpha.1
```

Please provide feedback in either the pinned issue or the discussion thread 🧵
